index.html

```diff
@@ -20,60 +20,65 @@
+              Node-RED Con
+            </a>
         </div>
       </div>
-
+    </div>
+    <div class="h-full max-w-2xl mx-auto">
       <!--Main-->
-      <div class="container pt-16 sm:pt-24 mx-auto space-y-12 mb-24">
-        <div class="flex flex-row mx-auto justify-center items-center space-x-2 sm:space-x-10">
-          <div class="w-36 sm:w-56">
+      <div id="content" class="container px-8 pt-8 sm:pt-10 mx-auto space-y-12 pb-24">
+        <div class="flex flex-col mx-auto justify-center items-center space-x-2 sm:space-x-10">
+          <div class="w-24">
             <img src="https://nodered.org/about/resources/media/node-red-hexagon.svg" />
           </div>
-          <div class="text-white space-y-1 md:space-y-2">
-            <h1 class="text-2xl sm:text-4xl">Node-RED Con</h1>
+          <div class="text-white space-y-1 md:space-y-2 text-center">
+            <h1 class="text-2xl sm:text-4xl">Node-RED Con 2025</h1>
+            <h2 class="text-xl sm:text-l">Tuesday, November 4, 2025 | Online</h1>
           </div>
         </div>
 
-        <div class="mx-4 md:mx-auto items-center text-white max-w-2xl text-lg space-y-4">
+        <div class="mx-4 md:mx-auto items-center text-white max-w-2xl text-lg space-y-8">
+          <p class="text-justify">
+            Node-RED Con 2025 is here! This year's online conference is all about Node-RED's impact on industry. It's a day for our community to share expertise, demonstrate powerful solutions, and connect with the developers and engineers solving real-world industrial challenges.
+          </p>
+          <h2 class="text-xl text-center pt-4">Call for Papers Officially Open</h2>
+          <p class="text-justify">
+            Node-RED Con is built from community submissions, and the Call for Papers is now officially open! We're seeking passionate individuals to share their knowledge through compelling case studies, technical deep-dives, inspiring "fun" projects, or as expert panelists. If you have a story to tell, we want to hear from you.
+          </p>
+
+          <div class="text-center">
+            <a class="inline-block p-6 mt-6 border border-white rounded-md bg-red-800 text-2xl hover:bg-red-700" target="_blank" href="https://www.papercall.io/node-red-con-2025">Submit Your Proposal</a>
+          </div>
+
+          <h2 class="text-xl text-center pt-4">Be the First to Know!</h2>
           <p class="text-justify">
-            Update: we have made the difficult decision to postpone the English Track of the 2023 Event. We have
-            a great line-up of talks, but some late logistical challenges meant we wouldn't be able to put on the
-            event we had hoped for. We are going to take a moment and replan for 2024.
+            We will annouce through the Node-RED social channels when registration opens for the event, but if you want to be notified, you can sign up <a href="">here</a> and our event sponsor, FlowFuse, will send you an email.
```

Comment:
Let's use this one for now, it can be published with this url, but I'll update it once we publish the blog article.
```suggestion
            We will annouce through the Node-RED social channels when registration opens for the event, but if you want to be notified, you can sign up <a href="https://share-eu1.hsforms.com/17D4mAX30S4-J_FJE2j23Mgftty7">here</a> and our event sponsor, FlowFuse, will send you an email.
```